src/range/Ranges.js

```diff
@@ -140,13 +166,23 @@ class Ranges extends Array {
         return this;
     }
 
+    /**
+     * This function return the peak list as a object with x and y arrays
+     * @param {Object} opt - see the options parameter in peak2vector function documentation
```

Comment:
try to add a referenc to the peak2vector function. Something like `{@link#peak2vector}` ?